docs/public/10-repo-setup.rst

```diff
@@ -164,36 +182,71 @@ License
 
 Open Source Cosmoz components use the Apache-2.0 license.
 
-This should be set/present in:
+This should be set or present in the files noted below.
+
+:file:`.bower.json`
+~~~~~~~~~~~~~~~~~~~
 
-* ``bower.json``
-* ``package.json``
+The Bower package manager configuration, add a license property with as the value set to the Apache 2.0 license. to this file.
 
-Also, a ``LICENSE`` file containing the Apache 2.0 License should be present in the repository root.
+Ommit the ``{`` and ``}`` if the file already has these.
```

Comment:
Omit

Reply:
Corrected to Omit on 2 places in the file.